robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentingClassLoader.java

```diff
@@ -649,24 +620,25 @@ private void instrumentNormalMethod(MethodNode method) {
       delegatorMethodNode.visibleAnnotations = method.visibleAnnotations;
       delegatorMethodNode.access &= ~(ACC_NATIVE | ACC_ABSTRACT | ACC_FINAL);
 
-      makePrivate(method);
+      makeMethodPrivate(method);
 
-      MyGenerator m = new MyGenerator(delegatorMethodNode);
+      RobolectricGeneratorAdapter generator = new RobolectricGeneratorAdapter(delegatorMethodNode);
 
-      generateShadowCall(method, originalName, m);
+      generateShadowCall(method, originalName, generator);
 
-      m.endMethod();
+      generator.endMethod();
 
       classNode.methods.add(delegatorMethodNode);
     }
 
+    //todo rename
```

Comment:
It's a todo to rename the method. It should be a verb, but I haven't got to figure out the purpose of this yet